src/components/tasks/TasksContent.tsx

```diff
@@ -40,11 +51,22 @@ export const TasksContent = () => {
 
     if (isLoading) return <p>Loading...</p>;
 
+    console.log(typeof dev);
     return (
         <div className={classNames.tasksContainer}>
             <TabSection onSelect={onSelect} activeTab={activeTab} />
             <div>
-                {tasksGroupedByStatus[activeTab] ? (
+                {dev === 'true' ? (
+                    tasks && tasks.length ? (
+                        <TaskList
+                            tasks={tasks}
+                            isEditable={isEditable}
+                            updateCardContent={updateCardContent}
+                        />
+                    ) : (
+                        <p>{NO_TASKS_FOUND_MESSAGE}</p>
+                    )
+                ) : tasksGroupedByStatus[activeTab] ? (
```

Comment:
sunny can you please convert this into if else condition to make it more readable, in my previous PR I was suggested to do that and it worked and made the code look readable, so I request you to see into it if it can be done.

Reply:
I think not because it is only a condition. Generally, if we have to check multiple conditions then we use if-else.

Reply:
I do agree with @Pratiyushkumar here. 
There are multiple cases here
1. check dev or not 
2. If it's dev, there is a task or not?
3. It it's not a dev, then show tasksGroupedByShow 

An if-else block would improve the readability.
That being said in this particular case readability is not hampered by that much so you may choose to ignore this for now.